src/views/Hooks/index.jsx

```diff
@@ -0,0 +1,39 @@
+import { hot } from 'react-hot-loader';
+import { Component } from 'react';
+import { Switch } from 'react-router-dom';
+import RouteWithProps from '../../components/RouteWithProps';
+import loadable from '../../utils/loadable';
+
+const ViewHooks = loadable(() =>
+  import(/* webpackChunkName: 'Clients.ViewHooks' */ './ViewHooks')
```

Comment:
Can we make this one `Hooks.ListHooks` for a `ListHooks` so it's a little more distinct from the single hook page?